common/src/main/scala/domain/api/request/CreateCompanyRequest.scala

```diff
@@ -0,0 +1,12 @@
+package domain.api.request
+
+import upickle.default._
+
+case class CreateCompanyRequest(
+    name: String,
+    url: String
+)
+
+object CreateCompanyRequest{
+  implicit val rw: ReadWriter[CreateCompanyRequest] = macroRW
```

Comment:
What's this? Does it generate "serdes"/ReadWriters for the type you require?

Reply:
Correct! It's the json encoder/decoders for upickle. 

A while ago, zio-json wasn't working for some reason on the scalajs portion of the code (at least, the codec generation) so I swapped it for upickle. 

Hoping to see if that issue is gone now after some updates, and switch back for consistency.